autoload/neomake/makers/ft/typescript.vim

```diff
@@ -9,19 +9,14 @@ function! neomake#makers#ft#typescript#tsc() abort
     return {
         \ 'args': ['--project', neomake#utils#FindGlobFile(expand('%:p:h'), 'tsconfig.json'), '--noEmit'],
         \ 'append_file': 0,
-        \ 'errorformat':
-            \ '%E%f %#(%l\,%c): error %m,' .
-            \ '%E%f %#(%l\,%c): %m,' .
-            \ '%Eerror %m,' .
-            \ '%C%\s%\+%m'
+        \ 'errorformat': '%f(%l\,%c): %m'
```

Comment:
So there might be something else: try using `let g:neomake_logfile = '/tmp/neomake.log'` and see what's going on really by `tail -f`ing it.